lib/asset/asset.js

```diff
@@ -13,7 +13,8 @@
 "use strict";
 
 require("core-js/stable");
-const { dirname: urlPathDirname, basename: urlPathBasename } = require("path").posix;
+const { basename: filePathBasename } = require("path");
```

Comment:
@mfrisbey I picked `posix` with the thought that it would behave the same whether it runs on Windows or Mac. Technically on Windows the path separator is `\` and not `/` which is the separator in urls. It looks like you already verified this though.